howdoi/howdoi.py

```diff
@@ -410,6 +446,8 @@ def get_parser():
     parser.add_argument('-n', '--num-answers', help='number of answers to return', default=1, type=int)
     parser.add_argument('-C', '--clear-cache', help='clear the cache',
                         action='store_true')
+    parser.add_argument('-j', '--json-output', help='return answers in raw json format',
```

Comment:
I like `-j` but I was looking around at some other repos and it seems like `-o json` is also somewhat common. Kubernetes uses it for `kubectl`, as done this [feature request](https://github.com/helm/helm/issues/1534). However I don't think we would ever support `-o yaml` so let's stick with `-j` for now.